test/browser/index.js

```diff
@@ -178,11 +178,19 @@ async function renderSource(svg) {
 		svgText = await response.text();
 	}
 
-	const parser = new Parser();
-	const document = parser.parseFromString(svgText);
+	try {
+		svgOutput.innerHTML = svgText;
+	}
+	catch {
+		svgOutput.innerHTML = '<svg>';
+		return;
+	}
 
-	svgOutput.innerHTML = '';
-	svgOutput.append(document.documentElement);
+	if (custom.resize.checked) {
+		const svgElement = svgOutput.childNodes[0];
+		svgElement.style.width = width;
+		svgElement.style.height = height;
```

Comment:
resizing the svg also ... this way you get a 1 to 1 match between left and right (look at hello world)